### ISSUE TITLE
Being able to undo a vote

### ISSUE THREAD

**Issue by heliosam** [retro-tool/retro-tool#19]
Maybe you change your mind and you want to put one of your votes in a different card: currently if you click on the one you just clicked, what happens is that you give an extra vote to it, all the contrary you were trying to do.

**Comment by uesteibar** [comment]
I think unvoting would be a very useful feature, this will need some design @iamsilesc

**Comment by rudeayelo** [comment]
This is a recurring topic related with limiting the number of votes and only being able to vote once for each item, should we reopen that discussion?

**Comment by uesteibar** [comment]
I would reopen each of those discussions separately, since they are complementary. Can you open/reopen issues to discuss those two other topics @rudeayelo?

**Comment by heliosam** [comment]
I remember discussing this with @uesteibar and @marciobarrios and it is true that in the end it's up to the team to set the rules they want to use in the retro (of course you cannot control that, but it is a matter of trust).

My 2 cents on these are:
- It is cool to vote more than once to the same card.
- There's no need to limit it; BUT setting the maximum number of votes while transitioning to the voting phase shouldn't harm a lot the UX (input field next to the button? small popup with the input field?)

Why I see being able to un-vote is because you can read other card after you already used your (trust or hard limited) number of votes or if you do it by mistake.

**Comment by uesteibar** [comment]
> Why I see being able to un-vote is because you can read other card after you already used your (trust or hard limited) number of votes or if you do it by mistake.

Yep, this is mainly why I see the unvote as something we should implement.

---

> It is cool to vote more than once to the same card.

Indeed, I wouldn't go into limiting to one vote per item. We can revisit it in the future if it happens to be a common request.

---

> setting the maximum number of votes while transitioning to the voting phase shouldn't harm a lot the UX

I feel like this is going to be a long discussion so let's move it to it's own issue and leave this one to discuss the _unvote_ feature: https://github.com/retro-tool/retro-tool/issues/20
